client/client.go

```diff
@@ -92,7 +93,13 @@ func (c *Client) ReceivedVouchers() <-chan payments.Voucher {
 }
 
 // CreateVirtualChannel creates a virtual channel with the counterParty using ledger channels with the intermediary.
-func (c *Client) CreateVirtualChannel(objectiveRequest virtualfund.ObjectiveRequest) virtualfund.ObjectiveResponse {
+func (c *Client) CreateVirtualChannel(request virtualfund.ObjectiveRequestForVirtualPaymentApp) virtualfund.ObjectiveResponse {
```

Comment:
Maybe we should rename `CreateVirtualChannel` to `CreateVirtualPaymentChannel` to keep it in line with `ObjectiveRequestForVirtualPaymentApp`?

Reply:
[5284054](https://github.com/statechannels/go-nitro/pull/881/commits/528405467e497575796b3780399ec5398f3b60cc)